vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -422,6 +419,36 @@ public VimeoAccount authorizeWithClientCredentialsGrantSync() {
         return call;
     }
 
+    /**
+     * Synchronous version of {@link #singleSignOnTokenExchange(String, AuthCallback)}. This is useful
+     * when already running in a background thread, such as when using the Android AccountManager.
```

Comment:
Can you elaborate in these docs on the use case for this method? Can't hurt to get specific.